### ISSUE TITLE
[Browser Plugin Connector] Implement Connect request Popup in Status desktop

### ISSUE THREAD

**Issue by kounkou** [status-im/status-desktop#15171]
### Description

Part of Epic https://github.com/status-im/status-desktop/issues/14794

Implement the connect request popup in status desktop

### Acceptance criteria

- Popup is matches the Figma

### Figma

![Screenshot from 2024-06-13 13-28-38](https://github.com/status-im/status-desktop/assets/2589171/5a237ff0-2c49-435a-98de-61fe6b197657)

https://www.figma.com/design/1OYKMzU6KTQHQAqDhojk0r/Status-connector?node-id=2013-4435&t=txSUoRXz1fXillcv-0

**Comment by noeliaSD** [comment]
Hey!! I see this task as a duplication of this epic:  https://github.com/status-im/status-desktop/issues/14678

Please @jrainville review! If it's only to change the model data or add some ui particularities to the shared epic, please _block_ it and link with the previously shared epic!

**Comment by noeliaSD** [comment]
And @jrainville if it is for **2.30**, the `ui-team` doesn't have capacity for giving you support!

**Comment by jrainville** [comment]
Indeed, this ticket is a duplicate of https://github.com/status-im/status-desktop/issues/14688
though https://github.com/status-im/status-desktop/issues/14688 is made for walletconnect, in the end it's the same modal with a different icon.

I'll leave this issue open so we can track the small differences, but I'll mark this one as blocked by https://github.com/status-im/status-desktop/issues/14688
